sandbox/src/mocks/changeShortlistAndSendForTriage/requests/Parameters.json

```diff
@@ -0,0 +1,97 @@
+{
+    "resourceType": "Parameters",
+    "meta": {
+        "profile": [
+            "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-ChangeShortlistAndSendForTriageRequest-Parameters-1"
```

Comment:
For the other Request/Response parameters we have "eRS-SendCommunicationToRequester-Request-Parameters-1" / "eRS-SendCommunicationToRequester-Response-Parameters-1"

May be nice to have the same format? Provided URLs haven't been finalised already?